docs/how-to/create-simple-chat-app.md

```diff
@@ -35,9 +42,9 @@ The diagram above demonstrates what a three-user network can look like. It's wor
 
 ### Docker (optional)
 
-If you don't want to use Docker, skip to the [**WebRTC-Star**](#webrtc-star) section.
+Skip to the [**WebRTC-Star**](#webrtc-star) section, if you are not using Docker
```

Comment:
Looks like the "." got removed erroneously :)

Reply:
ooops